runtime/src/snapshot_utils/snapshot_storage_rebuilder.rs

```diff
@@ -31,6 +31,14 @@ use {
         time::Instant,
     },
 };
+
+lazy_static! {
+    static ref VERSION_FILE_REGEX: Regex = Regex::new(r"^version$").unwrap();
+    static ref BANK_FIELDS_FILE_REGEX: Regex = Regex::new(r"^[0-9]+(\.pre)?$").unwrap();
```

Comment:
Noting a minor difference: the added `(\.pre)`, for an optional ".pre" at the end of the bank fields file.

Reply:
It makes it possible to load bank-fields from `<slot>.pre` file if it were in the archive, instead of only `<slot>`.

We shouldn't have the .pre files in the archive, so this is probably fine as is.